### ISSUE TITLE
I can't find any example how to use `overrides`, and the way I use `overrides` in eslint doesn't seem to apply.

### ISSUE THREAD

**Issue by Boshen** [oxc-project/oxc#3491]
The link above no longer point to any example, may I have a new link? I can't find any example how to use `overrides`, and the way I use `overrides` in eslint doesn't seem to apply.

_Originally posted by @bigopon in https://github.com/oxc-project/oxc/issues/2714#issuecomment-2141847891_

**Comment by Boshen** [comment]
`overrides` is not implemented yet.

**Comment by ThatGuySam** [comment]
This is a hacky solution but I ended up creating 2 config files with 2 separate commands and just turned off the rules I didn't want: 


```json
{
    ...
    "scripts": {
        "oxl": "pnpm dlx oxlint@latest --allow correctness",
        "oxlint-src": "pnpm oxl --config ../../packages/eslint-config/oxlint.config.json ./src ./test ./netlify ./*.ts ./*.js",
        "oxlint-scripts": "pnpm oxl --config ../../packages/eslint-config/oxlint-scripts.config.json ./script",
        "oxlint": "pnpm oxlint-src && pnpm oxlint-scripts",
    }
    ...
}
```

Also, for some reason I couldn't get either `--ignore-pattern` or `--ignore-path` to work: 
```bash
pnpm dlx oxlint@latest --allow correctness --ignore-path ./scripts
```

**Comment by DonIsaac** [comment]
https://github.com/oxc-project/backlog/issues/87

**Comment by Boshen** [comment]
overrides will be supported in Oxlint Beta. https://github.com/oxc-project/oxc/issues/3491

Closing for house keeping.
